example/src/main/java/org/aerogear/mobile/example/ui/SecurityServiceFragment.java

```diff
@@ -86,11 +86,11 @@ public void runTests() {
         detectDeviceLock();
         detectEmulator();
         debuggerDetected();
-        detectHookingFramework();
-        detectBackupEnabled();
-        detectDeviceEncryptionStatus();
-        detectLatestOS();
-        detectDeveloperOptions();
+//        detectHookingFramework();
```

Comment:
do we really want do comment those out ? 
why not removing ? 

If we wanna keep them in the examples, let's add a comment